net/netfilter/nf_tables_api.c

```diff
@@ -8360,6 +8558,7 @@ static int __nf_tables_abort(struct net *net, enum nfnl_abort_action action)
 			list_del_rcu(&nft_trans_flowtable(trans)->list);
 			nft_unregister_flowtable_net_hooks(net,
 						nft_trans_flowtable(trans));
+			break;
```

Comment:
Interesting this has been missing its from code that has been there a long time:
https://github.com/ctrliq/kernel-src-tree/commit/3b49e2e94e6ebb8b23d0955d9e898254455734f8

Reply:
I was trying to do the right thing there.